temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -652,3 +653,95 @@ message GetUsageResponse {
     // The next page's token.
     string next_page_token = 2;
 }
+
+message CreateExportSinkRequest {
+    // The namespace under which the sink is configured.
+    string namespace = 1;
+    // The specification for the export sink.
+    temporal.api.cloud.sink.v1.ExportSinkSpec spec = 2;
+    // Optional. The ID to use for this async operation.
+    string async_operation_id = 3;
+}
+
+message CreateExportSinkResponse {
+    // The async operation.
+    temporal.api.cloud.operation.v1.AsyncOperation async_operation = 2;
+}
+
+message GetExportSinkRequest {
+    // The namespace to which the sink belongs.
+    string namespace = 1;
+    // The name of the sink to retrieve.
+    string sink_name = 2;
+}
+
+message GetExportSinkResponse {
+    // The export sink retrieved.
+   temporal.api.cloud.sink.v1.ExportSink sink = 1;
+}
+
+message GetExportSinksRequest {
+    // The namespace to which the sinks belong.
+    string namespace = 1;
+    // The requested size of the page to retrieve. Cannot exceed 1000.
+    // Defaults to 100 if not specified.
+    int32 page_size = 2;
+    // The page token if this is continuing from another response - optional.
+    string page_token = 3;
+}
+
+message GetExportSinksResponse {
+    // The list of export sinks retrieved.
+    repeated temporal.api.cloud.sink.v1.ExportSink sinks = 1;
+    // The next page token, set if there is another page.
+    string next_page_token = 2;
+}
+
+message UpdateExportSinkRequest {
```

Comment:
Just confirming we can handle the MRN case correctly.

Reply:
Yes. We could.